package.json

```diff
@@ -54,41 +54,48 @@
   },
   "license": "MIT",
   "dependencies": {
-    "babel-cli": "^6.24.0",
-    "babel-core": "^6.24.0",
-    "babel-loader": "^7.1.2",
-    "babel-plugin-transform-decorators-legacy": "^1.3.4",
-    "babel-preset-env": "^1.6.1",
-    "babel-preset-react": "^6.23.0",
-    "babel-preset-stage-1": "^6.22.0",
+    "@babel/cli": "^7.0.0",
+    "@babel/core": "^7.0.0",
+    "@babel/plugin-proposal-class-properties": "^7.0.0",
+    "@babel/plugin-proposal-decorators": "^7.0.0",
+    "@babel/plugin-syntax-dynamic-import": "^7.0.0",
+    "@babel/plugin-transform-destructuring": "^7.0.0",
+    "@babel/preset-env": "^7.0.0",
+    "@babel/preset-react": "^7.0.0",
+    "babel-loader": "^8.0.0",
     "builder": "^3.2.3",
     "exenv": "^1.2.1",
+    "hoist-non-react-statics": "3.3.0",
     "inline-style-prefixer": "^4.0.0",
     "prop-types": "^15.5.8",
     "publishr": "^1.0.0",
     "rimraf": "^2.6.1",
     "webpack": "^3.10.0"
   },
   "peerDependencies": {
-    "react": "^15.3.0 || ^16.0.0"
+    "react": "^16.8.0"
   },
   "devDependencies": {
-    "babel-eslint": "^7.1.1",
+    "@babel/core": "^7.0.0",
```

Comment:
this looks to be extraneous per @ryan-roemer's comment here https://github.com/FormidableLabs/radium/pull/1027#issuecomment-506441745 (included in non-dev deps)